tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -33,6 +36,8 @@ limitations under the License.
 
 namespace tensorflow {
 
+static const char* kS3FileSystemLoggingTag = "S3FileSystemLogging";
```

Comment:
Looks like much of this file should be in an anonymous namespace. Could you fix that?

Reply:
Thanks @drpngx. The anonymous namespace have been fixed.